lms/templates/widgets/segment-io-footer.html

```diff
@@ -0,0 +1,14 @@
+% if settings.LMS_SEGMENT_KEY:
+    <!-- begin segment footer -->
```

Comment:
The implementations are not the same though, so as much as I LOVE to remove code duplication, I'm inclined to leave this alone.